pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -689,44 +686,6 @@ func registerBackupFixtures(r registry.Registry) {
 	}
 }
 
-func maybePutAzureCredentialsFile(ctx context.Context, c cluster.Cluster, path string) error {
-	if c.Cloud() != spec.Azure {
-		return nil
-	}
-
-	type azureCreds struct {
-		TenantID     string `yaml:"azure_tenant_id"`
-		ClientID     string `yaml:"azure_client_id"`
-		ClientSecret string `yaml:"azure_client_secret"`
-	}
-
-	azureEnvVars := []string{AzureTenantIDEnvVar, AzureClientIDEnvVar, AzureClientSecretEnvVar}
```

Comment:
These environment variable definitions here were added in the original PR:

https://github.com/cockroachdb/cockroach/blob/4863e2d8d84d2ea68d6918dd4de9b7f5461b5588/pkg/cmd/roachtest/tests/backup.go#L68-L70

Maybe they can be removed too?